pages/common/at.md

```diff
@@ -0,0 +1,16 @@
+# at
+
+> Execute commands once at a later time.
+> \*Make sure the atd (or atrun) is running.
```

Comment:
It's ok to simply use two sentences, with the second one providing useful instructions. This is quite common actually -- see for example [pmset.md](https://github.com/tldr-pages/tldr/blob/master/pages/osx/pmset.md), [salt-key.md](https://github.com/tldr-pages/tldr/blob/master/pages/common/salt-key.md), and [lsof.md](https://github.com/tldr-pages/tldr/blob/master/pages/common/lsof.md).

As you can see, there isn't a rigid standard for such notes. What I could recommend in this case is perhaps using the passive voice, so that it feels more like a description than an instruction (i.e. "X should be done" rather than "you should do X").